### PR TITLE
Changelog for MLflow 1.3

### PR DESCRIPTION
**Pull request by smurching**
Adds changelog for MLflow 1.3

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,41 @@
 Changelog
 =========
+1.3 (2019-09-30)
+------------------
+MLflow 1.3.0 includes several major features and improvements:
+
+Features:
+
+- The Python client now supports logging & loading models using TensorFlow 2.0 (#1872, @juntai-zheng)
```
Comment (sueann) [inline CHANGELOG.rst]: w00t!

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- Dramatic performance improvements when fetching runs and experiments in MLflow servers that use SQL database-backed storage (#1767, #1878, #1805 @dbczumar)
+- New ``GetExperimentByName`` REST API endpoint, callable via the Python client (#1775, @smurching)
+- New ``mlflow.delete_run``, ``mlflow.delete_experiment`` fluent APIs in the Python client(#1396, @MerelTheisenQB)
+- New CLI command (``mlflow experiments csv``) to export runs of an experiment into a CSV (#1705, @jdlesage)
+- Directories can now be logged as artifacts via ``mlflow.log_artifact`` in the Python fluent API (#1697, @apurva-koti)
+- HTML and geojson artifacts are now rendered in the run UI (#1838, @sim-san; #1803, @spadarian)
+- Keras autologging functionality now works when using the ``fit_generator`` Keras API (#1757, @charnger)
```
Comment (sueann) [inline CHANGELOG.rst]: maybe `Keras autologging support for ``fit_generator`` Keras API`` ?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
 =========
+1.3 (2019-09-30)
+------------------
+MLflow 1.3.0 includes several major features and improvements:
+
+Features:
+
+- The Python client now supports logging & loading models using TensorFlow 2.0 (#1872, @juntai-zheng)
+- Dramatic performance improvements when fetching runs and experiments in MLflow servers that use SQL database-backed storage (#1767, #1878, #1805 @dbczumar)
+- New ``GetExperimentByName`` REST API endpoint, callable via the Python client (#1775, @smurching)
```
Comment (sueann) [inline CHANGELOG.rst]: i forget, does this make `set_experiment` (and possibly other python APIs) faster?

**Review by sueann — COMMENTED** [review]
just a few questions. cc @stbof

File: CHANGELOG.rst (hunk 1 above)
Comment (smurching) [reply CHANGELOG.rst]: [just in time too](https://github.com/tensorflow/tensorflow/releases/tag/v2.0.0) :-)  cc @juntai-zheng

File: CHANGELOG.rst (hunk 3 above)
Comment (smurching) [reply CHANGELOG.rst]: yep, will mention this (good idea to mention the actual user impact)

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,5 +1,41 @@
 Changelog
 =========
+1.3 (2019-09-30)
+------------------
+MLflow 1.3.0 includes several major features and improvements:
+
+Features:
+
+- The Python client now supports logging & loading models using TensorFlow 2.0 (#1872, @juntai-zheng)
+- Dramatic performance improvements when fetching runs and experiments in MLflow servers that use SQL database-backed storage (#1767, #1878, #1805 @dbczumar)
```
Comment (mateiz) [inline CHANGELOG.rst]: Not sure whether "dramatic" is the right word, maybe it should be "significant"

**Comment by mateiz** [comment]
The notes look great to me overall; just left one comment.

**Pull request merged**